BraveWallet/Crypto/BuySwapSwap/SwapCryptoView.swift

```diff
@@ -7,8 +7,9 @@ import SwiftUI
 import BraveCore
 import BraveUI
 import struct Shared.Strings
+import Storage
 
-private struct RroundedBackgroundModifier: ViewModifier {
+struct RoundedBackground: View {
```

Comment:
If we're going to make this non-private, lets:
1. Move it to another file
2. Make its name more obvious for what it is for: `GroupedInsetRoundedBackgroundView` or `BuySendSwapGridBackgroundView` or something